### PR TITLE
Currency:Added build-script and build_info.json for directory-ldap-api

### PR DESCRIPTION
**Pull request by Pratikt2312**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: d/directory-ldap-api/directory-ldap-api_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# update the path env. variable
+export PATH=$PATH:$M2_HOME/bin
+
+# clone the repository
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+# Build 
+if ! mvn install; then
```
Comment (kandarpamalipeddi) [inline d/directory-ldap-api/directory-ldap-api_ubi_8.5.sh:53]: As you wanted to do only build, can you make this command to `mvn install -DskipTests` ?

Comment (Pratikt2312) [reply d/directory-ldap-api/directory-ldap-api_ubi_8.5.sh]: Updated.

**Comment by nikhil-kalbande** [comment]
@Pratikt2312 Can we move this script and folder to the format we intend to follow for all apache packages like `apache-directory-ldap-api`

File: d/directory-ldap-api/build_info.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "package_name": "directory-ldap-api",
+    "github_url": "https://github.com/apache/directory-ldap-api",
+    "version": "2.1.2",
+    "default_branch": "master",
+    "package_dir": "d/directory-ldap-api",
```
Comment (nikhil-kalbande) [inline d/directory-ldap-api/build_info.json:6]: @Pratikt2312 Can we move this script and folder to the format we intend to follow for all apache packages like apache-directory-ldap-api

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: d/directory-ldap-api/build_info.json (hunk 2 above)
Comment (Pratikt2312) [reply d/directory-ldap-api/build_info.json:6]: yes, sure.

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**